exercises/all-your-base/all-your-base-test.exs

```diff
@@ -0,0 +1,115 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("example.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule AllYourBaseTest do
+  use ExUnit.Case
+
+  test "convert single bit one to decimal" do
+    assert AllYourBase.convert([1], 2, 10) == [1]
+  end
+
+  @tag :pending
+  test "convert binary to single decimal" do
+    assert AllYourBase.convert([1, 0, 1], 2, 10) == [5]
+  end
+
+  @tag :pending
+  test "convert single decimal to binary" do
+    assert AllYourBase.convert([5], 10, 2) == [1, 0, 1]
+  end
+
+  @tag :pending
+  test "convert binary to multiple decimal" do
+    assert AllYourBase.convert([1, 0, 1, 0, 1, 0], 2, 10) == [4, 2]
+  end
+
+  @tag :pending
+  test "convert decimal to binary" do
+    assert AllYourBase.convert([4, 2], 10, 2) == [1, 0, 1, 0, 1, 0]
+  end
+
+  @tag :pending
+  test "convert trinary to hexadecimal" do
+    assert AllYourBase.convert([1, 1, 2, 0], 3, 16) == [2, 10]
+  end
+
+  @tag :pending
+  test "convert hexadecimal to trinary" do
+    assert AllYourBase.convert([2, 10], 16, 3) == [1, 1, 2, 0]
+  end
+
+  @tag :pending
+  test "convert 15-bit integer" do
+    assert AllYourBase.convert([3, 46, 60], 97, 73) == [6, 10, 45]
+  end
+
+  @tag :pending
+  test "convert empty list" do
+    assert AllYourBase.convert([], 2, 10) == nil
```

Comment:
`nil` agrees with the current canonical implementation. I think we will stick with that.